py/desitarget/secondary.py

```diff
@@ -346,26 +345,28 @@ def add_primary_info(scxtargs, priminfodir):
         primtargs = np.concatenate([primtargs, prim])
 
     # ADM make a unique look-up for the target sets.
-    scxbitnum = np.log2(scxtargs["SCND_TARGET"]).astype('int')
+    scxbitnum  = np.log2(scxtargs["SCND_TARGET"]).astype('int')
```

Comment:
In an number of places you introduce some extra spaces that don't conform to `PEP 8` code style. I'll probably catch these and fix them in a later PR, but if you want to fix them yourself you can see any issues in the [code style unit tests for your PR](https://travis-ci.org/desihub/desitarget/jobs/591940611#L973-L990).

Reply:
Bah, force of habit when parsing other people's code, sorry. Turns out my personal preference isn't in line (pun intended) with PEP8. Now I know, will fix.